src/main/java/com/conveyal/gtfs/validator/MissingStopCoordinatesValidator.java

```diff
@@ -16,12 +16,11 @@ public boolean validate(GTFSFeed feed, boolean repair) {
         boolean isValid = true;
         for (Stop stop : feed.stops.values()) {
             Coordinate stopCoord = new Coordinate(stop.stop_lat, stop.stop_lon);
-
-            ProjectedCoordinate projectedStopCoord = null;
             try {
-                projectedStopCoord = GeoUtils.convertLatLonToEuclidean(stopCoord);
+                // attempt to project coordinate
+                GeoUtils.convertLatLonToEuclidean(stopCoord);
```

Comment:
We shouldn't be using this function anywhere anymore; it attempts to project points to an "appropriate" UTM zone. Also, we're reversing lat and lon above (I think it get reversed again inside this function).